benchmark/child_process/spawn-echo.js

```diff
@@ -1,12 +1,12 @@
 'use strict';
 var common = require('../common.js');
 var bench = common.createBenchmark(main, {
-  thousands: [1]
+  len: [1000]
```

Comment:
Yeah I think I renamed this to `len` because I confused this one with another benchmark when editing :S..this should've been `n`